index.html

```diff
@@ -325,6 +325,79 @@ <h4>Connect your headset to your computer</h4>
+                <div class="row gx-4 gx-lg-5 justify-content-center">
+                    <div class="col-lg-8">
+                        <h2 class="text-black mb-4">Settings</h2>
+
+                        <div class="row gx-0 my-5 justify-content-center">
+                            <h4>Settings guide</h4>
+
+                            <ul class="nav nav-tabs" id="myTab2" role="tablist">
+                                <li class="nav-item" role="presentation">
+                                    <button class="nav-link active" id="foveation-tab" data-bs-toggle="tab" data-bs-target="#foveation" type="button" role="tab" aria-controls="adb" aria-selected="true">Foveation</button>
+                                </li>
+                                <li class="nav-item" role="presentation">
+                                    <button class="nav-link" id="bitrate-tab" data-bs-toggle="tab" data-bs-target="#bitrate" type="button" role="tab" aria-controls="bitrate" aria-selected="true">Bitrate</button>
+                                </li>
+                                <li class="nav-item" role="presentation">
+                                    <button class="nav-link" id="enconders-tab" data-bs-toggle="tab" data-bs-target="#encoders" type="button" role="tab" aria-controls="encoders" aria-selected="false">Encoders</button>
+                                </li>
+                            </ul>
+                            <div class="tab-content" id="myTabContent2">
+                                <div class="tab-pane fade show active" id="foveation" role="tabpanel" aria-labelledby="foveation-tab">
+                                    <p>A stronger foveation makes the image sharper in the center than in the periphery and makes the decoding faster. This is better for fast paced games. </p>
+
+                                    <p>A weaker foveation gives a uniform sharpness in the whole image.</p>
+                                        
+                                    <p>The recommended values are between 20% and 50% for headsets without eye tracking and between 50% and 70% for headsets with eye tracking.</p>
+                                </div>
+                                <div class="tab-pane fade" id="bitrate" role="tabpanel" aria-labelledby="bitrate-tab">
+                                    <p>Bitrate of the video, in Mbit/s.. Split among decoders based on size and codecs.</p>
+                                    <p>A higher bitrate directly improves image quality by reducing pixelation in fast-paced scenes or in complex visuals, such as landscapes with dense foliage. However, it may also increase latency or overload the network, encoder, or decoder.</p>
+                                    <p>For testing purposes, a bitrate of 50 Mbps is a good starting point for wireless connections. Wired connections, if supported by the hardware, should be able to handle up to 500 Mbps.</p>
```

Comment:
On dashboard we actually limit to 200.

Wired may have lower limits when using USB 2.0, 480Mbit/s is the theoretical link limit, actual transfer speed are lower.